telephony/android_modem.c

```diff
@@ -3290,163 +3307,165 @@ hasWaitingCall( AModem  modem )
 static const char*
 handleHangup( const char*  cmd, AModem  modem )
 {
-    if ( !memcmp(cmd, "+CHLD=", 6) ) {
-        int  nn;
-        cmd += 6;
-        switch (cmd[0]) {
-            case '0':  /* release all held, and set busy for waiting calls */
+    if ( memcmp(cmd, "+CHLD=", 6) ){
```

Comment:
Seriously, if you know these lines are going to be merged to __b2g-jellybean__, __b2g-kitkat__, and __b2g-lollipop__ branches, I guess you don't really want to change indentation of so many lines. And, __PLEASE DO NOT__ change line indentation along with other fixes at the same commit. Separate into two commits and make your change more clearer if fixing indentation is really necessary.